grammar.js

```diff
@@ -550,6 +552,8 @@ module.exports = grammar({
 
 		spread_expression: $ => prec.left(PREC.SPREAD, seq("*", $._expression)),
 
+		expect_expression: $ => prec.left(PREC.EXPECT, seq("expect", $.call_suffix)),
```

Comment:
In the official grammar they instead extend the IdentifierOrSoftKey rule to allow "expect", but maybe
this is fine for now like this.
Let's see how much parsing % we get with that.

Reply:
ah I see! doing this first and then moving to identifierorsoftkey sounds good to me 👍